datar/misc.py

```diff
@@ -1,3 +1,48 @@
+from typing import Any, Callable
+
+from pipda import register_verb
+
```

Comment:
Any, Callable, register_verb are polluting locals(), use as with '_' prefix for those imported variables